Zend/zend_hash.c

```diff
@@ -2270,7 +2270,13 @@ ZEND_API zval* ZEND_FASTCALL zend_hash_find(const HashTable *ht, zend_string *ke
 	return p ? &p->val : NULL;
 }
 
-ZEND_API zval* ZEND_FASTCALL _zend_hash_find_known_hash(const HashTable *ht, zend_string *key)
+ZEND_API ZEND_ATTRIBUTE_DEPRECATED
+zval* ZEND_FASTCALL _zend_hash_find_known_hash(const HashTable *ht, zend_string *key)
+{
+	return zend_hash_find_known_hash(ht, key);
+}
```

Comment:
You can just drop the old name.

Reply:
Okay, I'll drop the old name. However, I didn't change the API -- it was already `ZEND_API` in PHP 7.3: https://github.com/php/php-src/blob/PHP-7.3/Zend/zend_hash.h#L172.

Reply:
The underscore indicates that this is a private API that is exported for technical reasons.

Reply:
Okay. I don't see any reason not to export it -- in fact, `zend_hash_find_ex(..., 1)` was the most common way `zend_hash_find_ex` was used. Seems like a pretty important API.